tests/helpers/perf_run_helper.rb

```diff
@@ -549,9 +522,9 @@ def copy_scale_results(scenario)
 
     # copy puppet-metrics-collector to scale results dir (this iteration) and parent dir (entire scale run)
     # TODO: rename dir to 'puppet-metrics-collector'
```

Comment:
Delete this line

Reply:
I had thought about it but still wasn't sure if we might want to copy the entire thing for some reason. I've now removed it since we can always go back and re-add it.